lib/options.js

```diff
@@ -84,6 +77,40 @@ module.exports = optionator({
       type: 'Object',
       description: 'Specify rules'
     },
+
+    {
+      heading: 'Fixing problems'
+    },
+    {
+      option: 'fix',
+      type: 'Boolean',
+      default: 'false',
+      description: 'Automatically fix problems'
+    },
+    {
+      option: 'fix-to-stdout',
+      type: 'Boolean',
+      default: 'false',
+      description: 'Print the fix results to stdout '
+        + 'instead of regular output, must be used with --stdin'
+    },
+    {
+      option: 'fix-dry-run',
+      type: 'Boolean',
+      default: 'false',
+      description: 'Automatically fix problems without'
+        + 'saving the changes to the file system'
+        + ' (required the ESLint@4.9.0 or later)'
+    },
+    {
+      option: 'fix-type',
+      type: '[String]',
+      default: 'false',
+      description: 'Specify the types of fixes to apply'
+        + '(problem, suggestion, layout)'
+        + ' (required the ESLint@5.9.0 or later)'
```

Comment:
Some lines are missing blanks here and it's the blanks are sometimes at the end of a line. Can we make this consistent?